Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/Controller.purs

```diff
@@ -683,17 +682,22 @@ eval OnResumeCallback state =
   else 
     case getValueToLocalNativeStore LOCAL_STAGE of
       "FindingQuotes" -> do
+        let secondsLeft = findingQuotesSearchExpired false
         case (methodArgumentCount "startLottieProcess") == 1 of
           true  -> do
-            let secondsLeft = findingQuotesSearchExpired false
-                findingQuotesProgress = 1.0 - (toNumber secondsLeft)/(toNumber (getSearchExpiryTime "LazyCheck"))
+            let findingQuotesProgress = 1.0 - (toNumber secondsLeft)/(toNumber (getSearchExpiryTime "LazyCheck"))
             if secondsLeft > 0 then
               void $ pure $ startLottieProcess lottieAnimationConfig {rawJson = "progress_loader_line", lottieId = (getNewIDWithTag "lottieLoaderAnimProgress"), minProgress = findingQuotesProgress, scaleType="CENTER_CROP"}
             else pure unit
           false -> pure unit
         case flowWithoutOffers WithoutOffers of
-          true  -> exit $ OnResumeApp state
```

Comment:
so we will not call `notifyEvent` after resume, instead will call when app is opened or we click on home right ?